patch/1.19.9/lua-resty-core-ngx_meta_lua_module.patch

```diff
@@ -0,0 +1,273 @@
+diff --git lib/resty/core/shdict.lua lib/resty/core/shdict.lua
+index dedf12c..575cfb9 100644
+--- lib/resty/core/shdict.lua
++++ lib/resty/core/shdict.lua
+@@ -1,7 +1,7 @@
+ -- Copyright (C) Yichun Zhang (agentzh)
+ 
+ 
+-local ffi = require 'ffi'
++local ffi = require "ffi"
```

Comment:
This modification doesn't seem to make sense

Reply:
revert this